### PR TITLE
[s]Remove false positive heavy modified client admin alert.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
There are too many edge cases and this covers none of them. also i think there is a bug relating to some power of 2 code or maybe some plane/multi-z code or something because these messages are common and admins are ignoring them.

This is totally not a ploy to get the bug fixed or the fix merged or whatever.

### CODE REVIEW

File: code/_onclick/ai.dm [hunk 1]
```diff
@@ -46,14 +46,6 @@
 	if(isnull(pixel_turf))
 		return
 	if(!can_see(A))
-		if(isturf(A)) //On unmodified clients clicking the static overlay clicks the turf underneath
-			return //So there's no point messaging admins
-		message_admins("[ADMIN_LOOKUPFLW(src)] might be running a modified client! (failed can_see on AI click of [A] (Turf Loc: [ADMIN_VERBOSEJMP(pixel_turf)]))")
-		var/message = "[key_name(src)] might be running a modified client! (failed can_see on AI click of [A] (Turf Loc: [AREACOORD(pixel_turf)]))"
-		log_admin(message)
-		if(REALTIMEOFDAY >= chnotify + 9000)
-			chnotify = REALTIMEOFDAY
```
Comment (tralezab) [inline code/_onclick/ai.dm:55]: Could you kill this variable while you're here, `chnotify`? Yes, this is its one use.

Comment (tralezab) [reply code/_onclick/ai.dm:55]: (it's in `code\modules\mob\living\silicon\ai\ai.dm`)

**Comment by LemonInTheDark** [comment]
>  i think there is a bug relating to some power of 2 code or maybe some plane/multi-z code or something because these messages are common and admins are ignoring them.
> 
> This is totally not a ploy to get the bug fixed or the fix merged or whatever.

oh no it's me

**Comment by Simplehorror** [comment]
why use [s] here?

**Comment by Mothblocks** [comment]
whenever i see this i poke the guy and ask if they have the glitch that lets them see through static and ask 'em to reconnect

probably not super useful

File: code/_onclick/ai.dm (hunk 1 above)
Comment (Mothblocks) [reply code/_onclick/ai.dm:55]: we have too many open prs so im just gonna merge this anyway u can web edit it later

**Pull request merged**